engine/frontend/src/component/ServiceInfo.js

```diff
@@ -61,32 +134,24 @@ const ServiceInfo = () => {
-                <div className='flex flex-col h-full space-y-1 bg-white'>
-                    <Heading content={`${enclaveName} ::${selected.name}`}/>
-                    <div className="flex-1">
-                        <div className="text-xl text-left h-fit mb-2 ml-5 text-black">
-                            Ports
-                        </div>
-                        <div className="overflow-auto">
-                            {
-                                selected.ports.map(port => {
-                                    const urlWithApplicationString = `${port.applicationProtocol}://localhost:${port.publicPortNumber}`
-                                    const urlWithoutApplicationString = `localhost:${port.publicPortNumber}`
-                                    const url = port.applicationProtocol ? urlWithApplicationString : urlWithoutApplicationString
+                <div className='flex flex-col h-full space-y-1 bg-[#171923]'>
+                    <Heading content={`${enclaveName} - ${selected.name}`}/>
+                    <TableContainer>
+                        <Table variant='simple' size='sm'>
+                            <Tbody>
+                                {tableRow("Name", () => selectedSerialized.name)}
+                                {tableRow("UUID", () => <pre>{selectedSerialized.serviceUuid}</pre>)}
+                                {tableRow("Status", () => statusBadge(selectedSerialized.serviceStatus))}
+                                {tableRow("Image", () => selectedSerialized.container.imageName)}
+                                {tableRow("Ports", () => codeBox(0, "ports", selectedSerialized.ports))}
+                                {tableRow("ENTRYPOINT", () => codeBox(1, "entrypoint", selectedSerialized.container.entrypointArgs))}
+                                {tableRow("CMD", () => codeBox(2, "cmd", selectedSerialized.container.cmdArgs))}
+                                {tableRow("ENV", () => codeBox(3, "env", selectedSerialized.container.envVars))}
+                            </Tbody>
+                        </Table>
+                    </TableContainer>
 
-                                    return (
-                                        <div className="h-fit flex flex-row space-x-10 ml-5 text-black">
-                                            <div> {port.portName}:</div>
-                                            <a href={url} rel="noreferrer" className="grow">
-                                                <u> {url} </u>
-                                            </a>
-                                        </div>
-                                    )
-                                })
-                            }
-                        </div>
-                    </div>
-                    <LogView heading={`Service Logs`} logs={logs}/>
+                    {/*<LogView heading={`Service Logs`} logs={logs}/>*/}
```

Comment:
@Peeeekay note that for now the logs are not showing on this page. So you'll need to move them to a new page.